lib/gssapi/krb5/sequence.c

```diff
@@ -188,7 +188,7 @@ _gssapi_msg_order_check(struct gss_msg_order *o, OM_uint32 seq_num)
     for (i = 0; i < o->length - 1; i++) {
 	if (o->elem[i] == seq_num)
 	    return GSS_S_DUPLICATE_TOKEN;
-	if (o->elem[i + 1] < seq_num && o->elem[i] < seq_num) {
```

Comment:
This has been broken since the original commit f24cb76b05 ("sequence number checks, order and replay")